pkgs/by-name/ca/catppuccin-whiskers/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+let version = "2.3.0";
+in rustPlatform.buildRustPackage {
+  pname = "catppuccin-whiskers";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "catppuccin";
+    repo = "whiskers";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-gxM2SZEc6RfV5XzSqCMxKqt7dPnsIIybl4B5bD66j6E=";
+  };
+
+  buildAndTestSubdir = "whiskers";
```

Comment:
This line shouldn't be needed anymore.

Reply:
Ah my bad. Noticed this earlier but forgot to remove